src/include/ipc4/base_fw.h

```diff
@@ -17,6 +17,10 @@
 #ifndef __SOF_IPC4_BASE_FW_H__
 #define __SOF_IPC4_BASE_FW_H__
 
+#ifdef CONFIG_INTEL
```

Comment:
Can we make this a new Kconfig as manifest data is useful and generic that could be used by anyone. e.g. ```KCONFIG_IPC4_MANIFEST_MODULES``` we can have other ipc4 manifest kconfisg too as I would imagine a lot of vendors would like to export data.